### PR TITLE
Configure heisenbridge entrypoint

### PR DESCRIPTION
**Pull request by mweinelt**
This change will put an executable in the bin/ output of the package
that can be put into the PATH variable for easier execution.

The `python -m heisenbridge` call will still work, because `main()` will automatically be called.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -19,6 +19,10 @@ install_requires =
 
 python_requires = >=3.6
 
+[options.entry_points]
+console_scripts =
+    heisenbridge = heisenbridge.__main__:main
```
Comment (antifuchs) [inline setup.cfg:24]: You can save the `__main__.py` reindentation by doing this, I have found:

```suggestion
    heisenbridge = heisenbridge.__main__
```

**Review by antifuchs — COMMENTED** [review]
Oh dang, I ran into this as well today, patched it in a slightly different way and was about to submit mine as a PR too. Adding my approach as a suggestion - take it if you like (-:

File: setup.cfg (hunk 1 above)
Comment (antifuchs) [reply setup.cfg:24]: oops, you can not do that after all. I'm not completely sure why my basic tests worked and the not-quite-basic ones now don't. Please disregard this suggestion!

**Comment by hifi** [comment]
LGTM, thanks!

**Pull request merged**